libs/common/src/helpers/date-validation.ts

```diff
@@ -27,6 +27,7 @@ type ErrorOrDate =
     }
   | {
       error: ValidationError;
+      parsedDate: undefined;
```

Comment:
I've added this return here so it's easier to do: 
`{ error, parsedDate } = getStandardDateValidationErrors(..)` in typescript files - of there's a better solution let me know!